compiler/control/OMROptions.cpp

```diff
@@ -475,6 +475,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
 #ifdef J9_PROJECT_SPECIFIC
    {"disableOSRGuardRemoval",             "O\tdisable OSR guard removal",                      TR::Options::disableOptimization, osrGuardRemoval, 0, "P"},
 #endif
+   {"disableOSRGuardsMerging",            "O\tdisable Merging of OSR guards", SET_OPTION_BIT(TR_DisableOSRGuardsMerging), "F"},
```

Comment:
To keep the naming consistent with the other OSR guard option on the line above, can you name this "disableOSRGuardMerging" instead?

Reply:
Renamed it in https://github.com/eclipse/omr/compare/187fb9a2da48d055c1502d7e6db121cc3a84cf34..83092473f37f365071a5f0739e9a4f76c2f10706